broker-cli/commands/wallet.js

```diff
@@ -529,6 +530,48 @@ async function unlock (args, opts, logger) {
   }
 }
 
+/**
+ * Recover a wallet from a provided mnemonic seed and password. backup file is optional
```

Comment:
[nit] More information about the backup file (i.e. off-chain funds) would be nice.